src/style/variables/colors.ts

```diff
@@ -0,0 +1,17 @@
+export const peachy = '#f65261';
```

Comment:
have you considered using theme https://github.com/cssinjs/theming? Together with typescript it works very good, the best feature of css in js I would say

Reply:
Yeah, I can imagine that. There is a build-in solution for theming in the jss package as I can see - https://cssinjs.org/react-jss?v=v10.4.0#theming
I just didn't have enough time to go deeper in all the features :)

Reply:
okay, just keep it in mind, maybe once you'll get some time for this 🙂